### PR TITLE
Get leverage brackets

### PR DESCRIPTION
**Pull request by Krisa**
A little special here as the response is depending on whether we send a symbol or not. All in all, we return at the end an array.

### CODE REVIEW

File: futures/mark_price.go [hunk 1]
```diff
@@ -115,3 +115,64 @@ type FundingRate struct {
...
+	r.setParam("symbol", s.symbol)
+	if s.symbol != "" {
+		r.setParam("symbol", s.symbol)
+	}
+	data, err := s.c.callAPI(ctx, r, opts...)
+	if err != nil {
+		return []*LeverageBracket{}, err
+	}
+
+	if s.symbol != "" {
```
Comment (adshao) [inline futures/mark_price.go]: We can use `common.ToJSONList` to forcedly convert data to a List, please refer to https://github.com/adshao/go-binance/blob/7314d5890dc68c918ef88ff5f2b5267dc2e8b154/futures/ticker_service.go#L32

**Comment by Krisa** [comment]
@adshao here we go

File: futures/mark_price_test.go [hunk 2]
```diff
@@ -124,3 +124,71 @@ func (s *fundingRateServiceTestSuite) assertFundingRateEqual(e, a *FundingRate)
...
+			"symbol": symbol,
+		})
+		s.assertRequestEqual(e, r)
+	})
+
+	res, err := s.client.NewGetLeverageBracketService().Symbol(symbol).Do(newContext())
+
+	s.r().NoError(err)
+
+	s.r().NoError(err)
```
Comment (adshao) [inline futures/mark_price_test.go]: It's a duplicated line to L163.

Comment (Krisa) [reply futures/mark_price_test.go]: Good catch. Removed.

**Pull request merged**